### PR TITLE
Write new tests

### PR DESCRIPTION
**Pull request by rubaiyat22**
Stack 🥞 (4/4)
Follows https://github.com/department-of-veterans-affairs/caseflow/pull/16520

### CODE REVIEW

File: spec/models/hearings/hearing_email_recipient_spec.rb [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+      context "invalid email validation" do
+        let(:email_address) { "blah" }
+
+        it "raises an error when email is invalid" do
+          expect { subject }
+            .to raise_error(ActiveRecord::RecordInvalid)
+            .with_message("Validation failed: Email address Validation failed: " \
+              "#{error_title} email does not appear to be a valid e-mail address"
+            )
```
Comment (tomas-nava) [inline spec/models/hearings/hearing_email_recipient_spec.rb:35]:
```suggestion
              "#{error_title} email does not appear to be a valid e-mail address")
```
linting

File: spec/models/hearings/hearing_email_recipient_spec.rb [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+        create(
+          :hearing_email_recipient,
+          :initialized,
+          :appellant_hearing_email_recipient
+        )
+      end
+      let(:roles) do
+        [
+          HearingEmailRecipient::RECIPIENT_ROLES[:appellant],
+          HearingEmailRecipient::RECIPIENT_ROLES[:veteran],
```
Comment (tomas-nava) [inline spec/models/hearings/hearing_email_recipient_spec.rb:89]:
```suggestion
          HearingEmailRecipient::RECIPIENT_ROLES[:veteran]
```
linting

**Review by tomas-nava — APPROVED** [review]
approve!

with non-blocking suggestions

File: spec/models/hearing_spec.rb [hunk 3]
```diff
@@ -164,4 +164,107 @@
...
+          appellant_email: appellant_email,
+          appellant_tz: appellant_tz,
+          judge_email: judge_email,
+          representative_email: representative_email,
+          representative_tz: representative_tz,
+          created_by: User.system_user
+        )
+      end
+
+      it "backfills virtual hearing data and returns recipient", :aggregate_failures do
```
[the remark this answers is not in the document]
Comment (rubaiyat22) [reply spec/models/hearing_spec.rb:191]: Added in this [commit](https://github.com/department-of-veterans-affairs/caseflow/pull/16528/commits/64849ed73ab12350e69790c7b2fe95b26f89c60a)

**Pull request merged**